lib/models/os.coffee

```diff
@@ -330,13 +342,25 @@ getOsModel = (deps, opts) ->
 	# 	fs.writeFile('foo/bar/config.json', JSON.stringify(config));
 	# });
 	###
-	exports.getConfig = (nameOrId, callback) ->
+	exports.getConfig = (nameOrId, options = {}, callback) ->
```

Comment:
Now that you have an optional argument, you do need to use `findCallback` to work out which argument was the callback. Currently `getConfig(123, function (config) { ... ` will try to use that callback function as the options, and blow up.